src/components/Flashcard.tsx

```diff
@@ -0,0 +1,170 @@
+import {
+    Text,
+    Card,
+    CardBody,
+    CardFooter,
+    Button,
+    Center,
+    useToast,
+    IconButton,
+    CardHeader,
+} from "@chakra-ui/react";
+import { DeleteIcon } from "@chakra-ui/icons";
+import IFlashcard from "../Interfaces/IFlashcard";
+import { useState } from "react";
+import axios from "axios";
+import config from "../../utils/config";
+
+interface FlashcardProps extends IFlashcard {
+    setChangeCardWatcher: React.Dispatch<React.SetStateAction<number>>;
+}
+
+export default function Flashcard({
+    card_id,
+    front,
+    back,
+    streak,
+    setChangeCardWatcher,
+}: FlashcardProps): JSX.Element {
+    const [side, setSide] = useState<"front" | "back">("front");
+
+    const toast = useToast();
+    const showToast = (
+        title: string,
+        description: string,
+        status:
+            | "info"
+            | "warning"
+            | "success"
+            | "error"
+            | "loading"
+            | undefined,
+        duration: number
+    ) => {
+        toast({
+            title: title,
+            description: description,
+            status: status,
+            duration: duration,
```

Comment:
Some of the previous toasts had hard coded durations but this one is a parameter